roles/lifecycle/tasks/main.yml

```diff
@@ -49,11 +49,17 @@
 - name: Place parameters.yml
   template:
     src: "{{ item }}.j2"
-    dest: "{{ lifecycle_current_release_symlink }}/app/config/{{ item }}"
+    dest: "{{ lifecycle_current_release_symlink }}/config/legacy/{{ item }}"
     mode: 0644
   with_items:
     - parameters.yml
 
+- name: Place .env.local file for local environment overrides\
```

Comment:
The release script does not leave the .env file in place. It is also in the .gitignore, so it is not cloned into the project when the tarbal is created.

The readme states that the .env.dist file can be used as a template for creating an .env file.

Please note that:
- the env distfile sets an app secret. You might want to ovewrite that one with something randomly generated?
- App debug is set to true by default, not something you want on prod.

Maybe we need to have a small pow wow on what we'd like to track in the repo, and what values should be overridden here?

Reply:
The secret is set is in the parameters.yml
As we discussed, only a .env file will be put in place now by Ansible